### PR TITLE
add: arrpc

### LINKED ISSUE #785: [Request] arrpc
**What software are you requesting to be packaged?**
https://github.com/OpenAsar/arrpc

**Describe the motivation**
A clear and concise description of why you want this package. Would this provide value to other users?

**Additional context**
https://docs.fedoraproject.org/en-US/packaging-guidelines/Node.js/#_example_spec

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

**Review by lleyton — CHANGES_REQUESTED** [review]
systemd macros?

**Review by korewaChino — DISMISSED** [review]
ok lgtm

**Comment by lleyton** [comment]
> nothing provides gcc-libs needed by arrpc-3.3.0-1.fc39.x86_64 from @commandline

File: anda/misc/arrpc/arrpc.spec [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+%install
+install -D -m755 arrpc %buildroot%_bindir/arrpc
+install -D -m644 %SOURCE1 %buildroot%_userunitdir/arrpc.service
+
+%post
+%systemd_user_post arrpc.service
+
+%preun
+%systemd_user_preun arrpc.service
+
```
Comment (lleyton) [inline anda/misc/arrpc/arrpc.spec:42]: if the service can be restarted, you should have a postun with systemd_user_postun_with_restart

**Review by lleyton — APPROVED** [review]

**Review by lleyton — CHANGES_REQUESTED** [review]
sorry, don't know why I did a approve

**Review by lleyton — APPROVED** [review]

**Pull request merged**